packages/genesys-spark-components/src/components/stable/gux-datepicker/gux-datepicker.tsx

```diff
@@ -377,7 +385,7 @@ export class GuxDatepicker {
       ) {
         this.typeYearValue(currentSectionValue, event.key);
       } else {
-        if (this.canSetDate(inputNumber)) {
+        if (this.canSetDate(inputNumber) && this.pressedKeys.size == 2) {
```

Comment:
This might not be necessary but what if we did `this.pressedKeys.size >= 2` as a fail safe just in case something happened and somehow pressedKeys was more than 2 at this point in the code during execution for whatever reason - it'd prob never happen or is very unlikely, but maybe something could get out of whack during program execution.

Reply:
Yeah I think this is a good idea and fixes and issue we were looking at where we were getting stuck in a broken state when trying to type `11`